app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java

```diff
@@ -4,16 +4,21 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
 import edu.wisc.ece.pinpoint.utils.FirebaseDriver;
 
 public class MainActivity extends AppCompatActivity {
     private FirebaseDriver firebase;
+    private BottomNavigationView navBar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         firebase = FirebaseDriver.getInstance();
+        navBar = findViewById(R.id.navBar);
+        navBar.getMenu().getItem(2).setEnabled(false);
```

Comment:
What exactly is this doing? According to the documentation it disallows the menu item from being invoked via its shortcut, but it remains visible.

Reply:
The Map button is not directly part of the menu. As such, the 4 menu buttons are typically poorly oriented around the map button. The best online solution was to create a central menu button (index 2) that is invisible. This code disables that invisible central button